src/status_im/translations/fr.cljs

```diff
@@ -117,15 +113,13 @@
    :confirmation-code                     (str "Merci ! Nous vous avons envoyé un message texte contenant le code "
                                                "de confirmation. Veuillez fournir ce code pour confirmer votre numéro de téléphone")
    :incorrect-code                        (str "Désolé, le code est incorrect. Veuillez le saisir à nouveau.")
-   :generate-passphrase                   (str "Je vais créer une phrase secrète pour vous permettre de restaurer votre "
-                                               "accès ou de vous connecter depuis un autre périphérique")
    :phew-here-is-your-passphrase          "*Ouf* c'était dur, voici votre phrase secrète, *écrivez-la et gardez-la en lieu sûr !* Vous en aurez besoin pour restaurer votre compte."
    :here-is-your-passphrase               "Voici votre phrase secrète, *écrivez-la et gardez-la en lieu sûr !* Vous en aurez besoin pour restaurer votre compte."
-   :written-down                          "Assurez-vous de l'avoir bien écrite"
+   :here-is-your-signing-phrase           "Voici votre phrase de signature. Vous l'utiliserez pour vérifier vos signatures. *Ecrivez la et conservez la soigneusement!*"
```

Comment:
I am not sure about `phrase de signature`. Maybe we can find something better?

Reply:
Indeed. Can you explain what is a signing phrase and what is it used for? Should help me find a better formulation.

Reply:
Maybe I can say `Voici votre phrase secrète pour vérifier les signatures`

Reply:
Let's stick with your current suggestion.